src/views/AnnualReport.vue

```diff
@@ -39,54 +39,89 @@
     />
 
     <StaffPaymentDialog
+      v-model:staffPaymentData="staffPaymentData"
```

Comment:
Please test this change.

If this doesn't work then we could use a value (prop) and event to keep staffPaymentData synchronized between THIS component and the StaffPaymentDialog sub-component.

Reply:
Unless I've already raised this issue? 😕 

If this _is_ a problem then there are several places that this substitution was made.

Reply:
OK, great great catch. 
![missing required prop](https://github.com/bcgov/business-filings-ui/assets/122301442/b3bd5ac3-ed6a-4f2c-9213-979d2d25f608)

I'm thinking of turning off the rule `vue/no-deprecated-v-bind-sync`. Moreover, I think this change was also done somewhere else. I'll be reverting this change everywhere.

Reply:
I think "deprecated" rules should be changed to warnings possible, since we'll eventually have to replace the deprecated feature by something new. (Or turned off with a comment in the config file.)

Reply:
When I set the rule to 'warn', `npm run lint` is changing them. I have the rule set to off in my latest commit. However, I'll look into fixing this without having the rule.

Reply:
I like the comment for the deprecated classes rule. Please add a similar comment for the deprecated sync that you turned off.

Reply:
Thanks for both comments :)